seseion13/src/com/itschool/course/Animal.java

```diff
@@ -0,0 +1,2 @@
+package com.itschool.course;public class Animal {
```

Comment:
these 2 (package and `Animal` class declaration) should be on different lines. Same for all other classes (`Cat`, `Dog` etc).
Also, why are they empty? Is it because you created them during the course but you were not able to keep up with the trainer? If so, remember that you can always watch again the course recording and go back here to reproduce what the trainer teaches (recommended).